types/kintone-utility/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for kintone-utility 0.4
+// Project: https://github.com/kintone/kintoneUtility
+// Definitions by: TAKAHASHI Shuuji <https://github.com/shuuji3>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/kintone-utility
+// TypeScript Version: 2.2
+
+export namespace rest {
```

Comment:
https://github.com/kintone/kintoneUtility/blob/master/src/js/main.js#L73 shows the library is exported as default and added to global. You want to make the shape correct. 
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
Also need test for used as global test